index.html

```diff
@@ -1,75 +1,130 @@
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
+    <div class='parent-container'>
+        <nav>
+            <img class="lambdalogo" src="/Users/asha/Documents/Sprint-Challenge--User-Interface/img/lambda-black.png" alt="Lambda Logo">
+            <a href='index.html'>Home</a>
+            <a href='about.html'>About</a>
+            <a href='#'>Products</a>
+            <a href='#'>Blog</a>
+            <a href='#'>Contact</a>
+        </nav>
+        <div class="container">
 
-        <section class="bottom-content">
+            <section class="top-content">
+                <div class='head-image'>
+                    <img class='second-image' src='/Users/asha/Documents/Sprint-Challenge--User-Interface/img/jumbo.jpg' alt='jumbo'>
+                </div>
+                <div class="text-container">
+                    <h2>The Future</h2>
+                    <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna
+                        eget finibus.</p>
+                </div>
+                <div class="text-container">
+                    <h2>The Past</h2>
+                    <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna
+                        eget finibus.</p>
+                </div>
+            </section>
 
-            <div class="text-container">
-                <h2>That's</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>No</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>Moon</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
+            <section class="middle-content">
 
-        </section>
-        
-        <footer>
-            <nav>
-                <a href="#">Home</a>
-                <a href="#">About</a>
-                <a href="#">Products</a>
-                <a href="#">Blog</a>
-                <a href="#">Contact</a>
-            </nav>
-        </footer>
-    </div><!-- container -->        
+                <h2>Why Did It Have To Be Boxes...</h2>
+
+                <div class="boxes">
+                    <div class='teal'>
+                        <div class="box">
```

Comment:
this works, but it's probably more common to put two class names on it, like we discussed last week.  Then there's no need to have nested divs one color and one box div.   Also make sure your css specificity is heavy enough so the styles take effect.